### ISSUE TITLE
julia documentation lookup seems broken for non-main packages 

### ISSUE THREAD

**Issue by dahtah** [emacs-ess/ESS#829]
Documentation lookup only works for functions in main, i.e. in the inferior julia process
```
?prod
```
works fine, but trying to look up anything not in main fails: 
```
using Pkg
?Pkg.add
```
gets me the following:
```
ERROR: UndefVarError: Pkg not defined
Stacktrace:
 [1] top-level scope at none:0
 [2] eval at ./boot.jl:319 [inlined]
 [3] help(::String) at ./none:18
 [4] top-level scope at none:0
```
It works fine in the standard Julia REPL. I'm using ESS 20190122.2108 from MELPA.

**Comment by vspinu** [comment]
But does it work  with `C-c C-d C-d`?

**Comment by dahtah** [comment]
Well, the following happens: I tried that on "neighbors" from package LightGraphs (NB: the word at point is just "neighbors", not "LightGraphs.neighbors"). The first hit on the list is "neighbors", which issues the following error in the mini-buffer: "No documentation found. Binding Main.ESS.neighbors does not exist". Further down the list there's LightGraphs.neighbors, which results in "ERROR: UndefVarError: LightGraphs not defined", this time in a new buffer, ie. the same as the error above. 
I should add that this is on Julia 1.0.

**Comment by vspinu** [comment]
I see. That means it's a deficiency of the help retrieval in [ess-julia.jl](https://github.com/emacs-ess/ESS/blob/master/etc/ess-julia.jl#L17-L19). We are relying on external contributors to fix broken parts in that file as julia evolves. Last time @nverno contributed and fixed quite a bit of stuff. Maybe he could help us with this one as well.

**Comment by jabranham** [comment]
Fixed in af0df03088ad076f6f5b8b97e552ed7b988f1bc2, see #834
